tests/benchmarks/model_benchmarks/test_pytorch_base.py

```diff
@@ -231,3 +229,34 @@ def test_pytorch_base():
     assert (isinstance(benchmark._optimizer, torch.optim.SGD))
     benchmark._optimizer_type = None
     assert (benchmark._create_optimizer() is False)
+
+    # Test _postprocess().
+    assert (benchmark._postprocess())
+
+
+@decorator.cuda_test
+@decorator.pytorch_test
+def test_pytorch_empty_cache():
+    """Test PytorchBase class."""
+    # Register mnist benchmark.
+    BenchmarkRegistry.register_benchmark('pytorch-mnist', PytorchMNIST)
+
+    # Test cache empty by manually calling torch.cuda.empty_cache().
+    parameters = '--batch_size 32 --num_warmup 8 --num_steps 64 --model_action train'
+    benchmark = PytorchMNIST('pytorch-mnist', parameters=parameters)
+    assert (benchmark)
+    assert (benchmark._preprocess())
+    assert (benchmark._benchmark())
+    del benchmark
+    assert (torch.cuda.memory_stats()['reserved_bytes.all.current'] > 0)
+    torch.cuda.empty_cache()
```

Comment:
why not run `benchmark._postprocess()` here?

Reply:
Here, all resources have been cleaned manually:
```
del benchmark
torch.cuda.empty_cache()
```
Don't need to call `_postprocess()` again.

Reply:
I mean this test case is used to check `_postprocess` will cleanup resource, so that before the fix the case would fail, with this fix the case will pass.

Reply:
The test function `test_pytorch_empty_cache()` is to test resource cleanup, not only for `_postprocess()`.
It has two sub-cases, one is to test the API `torch.cuda.empty_cache()` which is the one you pointed here. Another is to tested automatic resource cleanup in `launch_benchmark()`.

For `_postprocess()`, it is tested implicitly in the second sub-case.